api/src/main/java/org/openmrs/module/idgen/SequentialIdentifierGenerator.java

```diff
@@ -187,4 +192,17 @@ public Long getNextSequenceValue() {
 	public void setNextSequenceValue(Long nextSequenceValue) {
 		this.nextSequenceValue = nextSequenceValue;
 	}
+	
+	private PrefixProvider getPrefixProvider(String prefixProviderBean) {
+		return Context.getRegisteredComponent(prefixProviderBean, PrefixProvider.class);
+	}
+
+	public String getPrefixProviderBean() {
```

Comment:
Add a JavaDoc here, use this opportunity to expand on the chosen design of prefix providers.